test/integration/backward_compatible/parallel/map/NearCachedMapTest.js

```diff
@@ -179,6 +180,11 @@ describe('NearCachedMapTest', function () {
                 expectStats(map1, 0, 1, 0);
             });
 
+            it('removeAll clears nearcache', async function () {
+                this.map.removeAll(true);
+                this.assertEqual(0, this.map.wrapped.nearCache);
+            });
```

Comment:
`this.map` exists due to test runtime, mocha but you should not be using that. Our map proxies are map1 and map2.

let's think about what to test about removeAll on nearcachedproxy. There is one thing extra in nearcachedmapproxy compared to mapproxy, and that is clearing the near cache. So, let's test that.


 The test I wrote is inspired from `clear clears nearcache`:

```suggestion
            it('removeAll clears nearcache', async function () {
                await map1.get('key1');
                await map1.removeAll(Predicates.alwaysTrue());
                expectStats(map1, 0, 1, 0);
            });
```
Initially near cache is empty

1. With `await map1.get('key1');`, a cache miss happens. Also, entry key1 -> value1 is cached. So entry count is 1 here. 
2. we call removeAll, near cache is cleared.
3. with `expectStats` we assert that entry count in near cache is 0, meaning it is cleared.